internal/backuprestore/common.go

```diff
@@ -71,6 +73,19 @@ var (
 	restoreGvk = schema.GroupVersionKind{Group: "velero.io", Kind: "Restore", Version: "v1"}
 )
 
+// BackuperRestorer interface also used for mocks
+type BackuperRestorer interface {
+	GetSortedBackupsFromConfigmap(ctx context.Context, content []lcav1alpha1.ConfigMapRef) ([][]*velerov1.Backup, error)
+	StartOrTrackBackup(ctx context.Context, backups []*velerov1.Backup) (*BackupTracker, error)
+	LoadRestoresFromOadpRestorePath() ([][]*velerov1.Restore, error)
+	StartOrTrackRestore(ctx context.Context, restores []*velerov1.Restore) (*RestoreTracker, error)
+	ExportOadpConfigurationToDir(ctx context.Context, toDir, oadpNamespace string) error
+	ExportRestoresToDir(ctx context.Context, configMaps []lcav1alpha1.ConfigMapRef, toDir string) error
+	RestoreOadpConfigurations(ctx context.Context) error
+	CleanupBackups(ctx context.Context) (bool, error)
+	DeleteOadpOperator(ctx context.Context, namespace string) error
```

Comment:
nit: perhaps we can sort these methods alphabetically? This would also be consistent with the mockgen version...

Reply:
Updated